pages/app.tsx

```diff
@@ -7,7 +7,7 @@ import Link from 'next/link';
 
 export default function App() {
     const { user, isLoading } = useUser();
-    if (isLoading) { return <Spinner /> }
+    // if (isLoading) { return <Spinner /> }
```

Comment:
this should stay too. your temporary changes for testing should not be present in the final PR.